frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -229,7 +230,10 @@ export class ViewTitleHeader extends React.Component {
           )}
           {isRunnable && (
             <RunButtonWithTooltip
-              className={cx({ hidden: isShowingNotebook })}
+              className={cx("text-brand-hover", {
+                hidden: isShowingNotebook,
+                "text-white-hover": isDirty,
```

Comment:
I think this is in here so that the Play button in the top right will be white-on-blue, but this or something else is causing the refresh button to be white on light-gray on hover:

![image](https://user-images.githubusercontent.com/2223916/62890460-6392ff00-bcf8-11e9-82dd-b4add7fbf995.png)

Reply:
Should be fixed now